base/echo/echo-scheduler/profile.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
```

Comment:
It looks like these two `profile.yml` files aren't being referenced from the `kustomization`.yml` files, so probably aren't doing anything.

Reply:
🤦Great catch, I guess I deployed two over-achieving echos BOTH doing scheduling...